### PR TITLE
Add bash TUI functions to openshift scripts

### PR DESCRIPTION
**Pull request by devguyio**
This is the first of some UI enhancements for our midstream repos.

This adds a bunch of functions to be used in scripts to give a UX as the following

[asciicast](https://asciinema.org/a/450711)

### CODE REVIEW

File: openshift/common.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+# Colors
+COLOR_OFF='\033[0m'       # Text Reset
+RED='\033[0;31m'          # Red
+GREEN='\033[0;32m'        # Green
+YELLOW='\033[0;93m'        # Green
```
Comment (pierDipi) [inline openshift/common.sh:7]:
```suggestion
GREEN='\033[0;32m'        # Green
YELLOW='\033[0;93m'       # Yellow
```

File: openshift/tui-functions.sh [hunk 2]
```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+# Colors
+COLOR_OFF='\033[0m' # Text Reset
+RED='\033[0;31m'    # Red
+GREEN='\033[0;32m'  # Green
+YELLOW='\033[0;93m' # Yello
```
Comment (pierDipi) [inline openshift/tui-functions.sh:7]:
```suggestion
YELLOW='\033[0;93m' # Yellow
```

**Comment by devguyio** [comment]
@pierDipi done

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

Nice!

**Pull request merged**